FooBar.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+class FooBar
+{
+
+    // OLD RULES
+
+    public function foo($cash):string

```

Comment:
This function should not be public. Input argument `$cash` should have better name.